core/dungeon-masters-guide/items/items-wands.xml

```diff
@@ -281,4 +281,48 @@
 			<stat name="spellcasting:attack" value="3" />
 		</rules>
 	</element>
+	<element name="Wand of Orcus" type="Magic Item" source="Dungeon Master's Guide" id="ID_WOTC_DMG_MAGIC_ITEM_WAND_OF_ORCUS">
```

Comment:
Something is causing this and the Sword of Kas to be their own Source

Reply:
Different apostrophe because that's a thing